REPL is broken at `1.0.0-alpha14`

### Expected Behavior

Running `elide` in plain form should send the user to a JS REPL

### Actual Behaviour

Instead, an error surfaces in some contexts

### Stacktrace

```
Uncaught fatal exception: java.io.IOError: java.io.IOException: Error calling ioctl(TIOCGWINSZ): return code is -1
org.jline.terminal.impl.AbstractPosixTerminal.getSize(AbstractPosixTerminal.java:65)
org.jline.terminal.Terminal.getWidth(Terminal.java:253)
elide.tool.cli.cmd.repl.ToolShellCommand.buildTerminal(ToolShellCommand.kt:903)
elide.tool.cli.cmd.repl.ToolShellCommand.initCLI(ToolShellCommand.kt:947)
elide.tool.cli.cmd.repl.ToolShellCommand.beginInteractiveSession(ToolShellCommand.kt:1401)
elide.tool.cli.cmd.repl.ToolShellCommand.invoke$lambda$63$lambda$62(ToolShellCommand.kt:1976)
elide.tool.cli.AbstractSubcommand.withDeferredContext(AbstractSubcommand.kt:586)
elide.tool.cli.cmd.repl.ToolShellCommand.invoke(ToolShellCommand.kt:1947)
elide.tool.cli.cmd.repl.ToolShellCommand$invoke$1.invokeSuspend(ToolShellCommand.kt)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
elide.tool.cli.Elide.invoke-AZeniH0(Elide.kt:325)
elide.tool.cli.AbstractToolCommand$call$$inlined$execute$1.invokeSuspend(AbstractToolCommand.kt:185)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:55)
picocli.CommandLine.executeUserObject(CommandLine.java:2045)
picocli.CommandLine.access$1500(CommandLine.java:148)
picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
picocli.CommandLine.execute(CommandLine.java:2174)
elide.tool.cli.Elide$Companion.exec$cli(Elide.kt:258)
elide.tool.cli.Elide$Companion.entry(Elide.kt:220)
elide.tool.cli.MainKt.main(main.kt:131)
elide.tool.cli.MainKt$main$3.invoke(main.kt)
elide.tool.cli.MainKt$main$3.invoke(main.kt)
kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$$inlined$createCoroutineFromSuspendFunction$IntrinsicsKt__IntrinsicsJvmKt$1.invokeSuspend(IntrinsicsJvm.kt:223)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlin.coroutines.ContinuationKt.startCoroutine(Continuation.kt:115)
kotlin.coroutines.jvm.internal.RunSuspendKt.runSuspend(RunSuspend.kt:19)
elide.tool.cli.MainKt.main(main.kt)
java.base@23.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Uncaught fatal exception: java.io.IOError: java.io.IOException: Error calling ioctl(TIOCGWINSZ): return code is -1
org.jline.terminal.impl.AbstractPosixTerminal.getSize(AbstractPosixTerminal.java:65)
org.jline.terminal.Terminal.getWidth(Terminal.java:253)
elide.tool.cli.cmd.repl.ToolShellCommand.buildTerminal(ToolShellCommand.kt:903)
elide.tool.cli.cmd.repl.ToolShellCommand.initCLI(ToolShellCommand.kt:947)
elide.tool.cli.cmd.repl.ToolShellCommand.beginInteractiveSession(ToolShellCommand.kt:1401)
elide.tool.cli.cmd.repl.ToolShellCommand.invoke$lambda$63$lambda$62(ToolShellCommand.kt:1976)
elide.tool.cli.AbstractSubcommand.withDeferredContext(AbstractSubcommand.kt:586)
elide.tool.cli.cmd.repl.ToolShellCommand.invoke(ToolShellCommand.kt:1947)
elide.tool.cli.cmd.repl.ToolShellCommand$invoke$1.invokeSuspend(ToolShellCommand.kt)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
elide.tool.cli.Elide.invoke-AZeniH0(Elide.kt:325)
elide.tool.cli.AbstractToolCommand$call$$inlined$execute$1.invokeSuspend(AbstractToolCommand.kt:185)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:55)
picocli.CommandLine.executeUserObject(CommandLine.java:2045)
picocli.CommandLine.access$1500(CommandLine.java:148)
picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
picocli.CommandLine.execute(CommandLine.java:2174)
elide.tool.cli.Elide$Companion.exec$cli(Elide.kt:258)
elide.tool.cli.Elide$Companion.entry(Elide.kt:220)
elide.tool.cli.MainKt.main(main.kt:131)
elide.tool.cli.MainKt$main$3.invoke(main.kt)
elide.tool.cli.MainKt$main$3.invoke(main.kt)
kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$$inlined$createCoroutineFromSuspendFunction$IntrinsicsKt__IntrinsicsJvmKt$1.invokeSuspend(IntrinsicsJvm.kt:223)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
kotlin.coroutines.ContinuationKt.startCoroutine(Continuation.kt:115)
kotlin.coroutines.jvm.internal.RunSuspendKt.runSuspend(RunSuspend.kt:19)
elide.tool.cli.MainKt.main(main.kt)
java.base@23.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Exception in thread "main" java.io.IOError: java.io.IOException: Error calling ioctl(TIOCGWINSZ): return code is -1
        at org.jline.terminal.impl.AbstractPosixTerminal.getSize(AbstractPosixTerminal.java:65)
        at org.jline.terminal.Terminal.getWidth(Terminal.java:253)
        at elide.tool.cli.cmd.repl.ToolShellCommand.buildTerminal(ToolShellCommand.kt:903)
        at elide.tool.cli.cmd.repl.ToolShellCommand.initCLI(ToolShellCommand.kt:947)
        at elide.tool.cli.cmd.repl.ToolShellCommand.beginInteractiveSession(ToolShellCommand.kt:1401)
        at elide.tool.cli.cmd.repl.ToolShellCommand.invoke$lambda$63$lambda$62(ToolShellCommand.kt:1976)
        at elide.tool.cli.AbstractSubcommand.withDeferredContext(AbstractSubcommand.kt:586)
        at elide.tool.cli.cmd.repl.ToolShellCommand.invoke(ToolShellCommand.kt:1947)
        at elide.tool.cli.cmd.repl.ToolShellCommand$invoke$1.invokeSuspend(ToolShellCommand.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
        at elide.tool.cli.Elide.invoke-AZeniH0(Elide.kt:325)
        at elide.tool.cli.AbstractToolCommand$call$$inlined$execute$1.invokeSuspend(AbstractToolCommand.kt:185)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:291)
        at elide.tool.cli.AbstractToolCommand.call(AbstractToolCommand.kt:55)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at elide.tool.cli.Elide$Companion.exec$cli(Elide.kt:258)
        at elide.tool.cli.Elide$Companion.entry(Elide.kt:220)
        at elide.tool.cli.MainKt.main(main.kt:131)
        at elide.tool.cli.MainKt$main$3.invoke(main.kt)
        at elide.tool.cli.MainKt$main$3.invoke(main.kt)
        at kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$$inlined$createCoroutineFromSuspendFunction$IntrinsicsKt__IntrinsicsJvmKt$1.invokeSuspend(IntrinsicsJvm.kt:223)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlin.coroutines.ContinuationKt.startCoroutine(Continuation.kt:115)
        at kotlin.coroutines.jvm.internal.RunSuspendKt.runSuspend(RunSuspend.kt:19)
        at elide.tool.cli.MainKt.main(main.kt)
        at java.base@23.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Caused by: java.io.IOException: Error calling ioctl(TIOCGWINSZ): return code is -1
        at org.jline.terminal.impl.jansi.JansiNativePty.getSize(JansiNativePty.java:167)
        at org.jline.terminal.impl.AbstractPosixTerminal.getSize(AbstractPosixTerminal.java:63)
        ... 48 more
```

### Steps To Reproduce

N/A

### Environment Information

- Operating System: Linux
- Architecture: x86_64

### Runtime

Native

### Language

N/A (Default)

### Example Application

_No response_

### Version

1.0.0-alpha14

--- a/packages/cli/build.gradle.kts
+++ b/packages/cli/build.gradle.kts
@@ -139,7 +139,7 @@ val enableFfm = hostIsLinux && System.getProperty("os.arch") != "aarch64"
 val enableEmbeddedResources = false
 val enableResourceFilter = false
 val enableAuxCache = false
-val enableAuxCacheTrace = true
+val enableAuxCacheTrace = false
 val enableJpms = false
 val enableConscrypt = false
 val enableBouncycastle = false
@@ -1121,7 +1121,6 @@ val commonNativeArgs = listOfNotNull(
   "-H:ExcludeResources=elide/stream/.*\\.proto",
   "-H:ExcludeResources=elide/page/.*\\.proto",
   "-H:ExcludeResources=elide/vfs/.*\\.proto",
-  "-H:ExcludeResources=org/jline/.*",
   "-H:ExcludeResources=logback.xml",
   "-H:ExcludeResources=META-INF/native/libsqlite.*",
   "-H:ExcludeResources=lib/linux-x86_64/libbrotli.so",

--- a/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
+++ b/packages/cli/src/main/resources/META-INF/native-image/dev/elide/elide-cli/reachability-metadata.json
@@ -2918,7 +2918,15 @@
   ],
   "resources": [
     {
-      "glob": "sun/util/logging/resources/logging_en.properties"
+      "glob": "org/jline/utils/*.caps"
+    },
+    {
+      "glob": "sun/util/logging/resources/logging_en.properties",
+      "module": "java.logging"
+    },
+    {
+      "glob": "sun/util/logging/resources/logging_en_US.properties",
+      "module": "java.logging"
     },
     {
       "glob": "META-INF/maven/org.jline/jline-native/pom.properties"
@@ -3096,6 +3104,10 @@
     {
       "name": "com.sun.tools.javac.resources.compiler",
       "locales": ["en"]
+    },
+    {
+      "name": "sun.util.logging.resources.logging",
+      "locales": ["en", "en-US"]
     },
     {
       "name": "sun.tools.jar.resources.jar",

--- a/packages/graalvm/src/main/kotlin/elide/runtime/core/internals/graalvm/GraalVMEngine.kt
+++ b/packages/graalvm/src/main/kotlin/elide/runtime/core/internals/graalvm/GraalVMEngine.kt
@@ -244,12 +244,16 @@ import org.graalvm.polyglot.HostAccess as PolyglotHostAccess
     /** Whether to emit trace messages for aux cache usage. */
     private val traceCache = System.getProperty("elide.traceCache", "false") == "true"
 
+    /** Whether to use the auxiliary engine cache. */
+    private val enableAuxCache = System.getProperty("elide.auxCache", "false") == "true"
+
     /** Whether the auxiliary cache is actually enabled. */
     private val useAuxCache = (
       ENABLE_AUX_CACHE &&
+      enableAuxCache &&
       System.getProperty("elide.test") != "true" &&
       System.getProperty("ELIDE_TEST") != "true" &&
-      System.getProperty("elide.vm.engine.preinitialize") != "false" &&  // manual killswitch
+      System.getProperty("elide.vm.engine.preinitialize") != "false" &&  // manual kill-switch
       ImageInfo.inImageCode() &&
       !ImageInfo.isSharedLibrary() &&
       !Platform.includedIn(Platform.WINDOWS::class.java)  // disabled on windows - not supported

